cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -215,6 +215,37 @@ function apply_sriov_node_policy {
   return 0
 }
 
+function setns_sriov_interfaces {
```

Comment:
How about this name + args: `move_sriov_pfs_netns_to_node`, using as an argument the node name.

With this the comment in the next line can be dropped.